README.md

```diff
@@ -29,9 +29,9 @@ Alternatively, clone this repository and run `python setup.py install`.
 Node.js dependency
 ============
 
-[Node.js](https://nodejs.org/) is required to interpret Cloudflare's obfuscated JavaScript challenge.
+[Node.js](https://nodejs.org/) (at least 4.5) is required to interpret Cloudflare's obfuscated JavaScript challenge.
```

Comment:
This looks really great and informative but I do have a minor nick pick before I approve. Would you make that a tad more verbose? i.e. `(at least 4.5)` -> `(at least version 4.5)` or something similar.

I'm guessing people will assume that Node.js v4.5 is supported, however, there is no support for that version as it should not be used for security reasons. We'll want to keep it backwards compatible as much as possible but @Anorov would have to be asked whether or not he's willing to publicly state support for insecure versions of Node.js.

I'll approve this but I don't want to merge anything without explicit approval. Thanks for sending this PR.

Reply:
We can rephrase it, so that we actually suggest recent NodeJS releases, but note that <= 4.4 is definitely broken, if you prefer?

Reply:
Sounds good to me.

Reply:
Opening a new PR as I made a mess here.